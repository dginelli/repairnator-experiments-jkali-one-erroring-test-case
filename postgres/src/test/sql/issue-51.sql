select * from zdb_tally('so_posts', 'last_activity_date', 'week', 'cheese', 5000, 'term');
select * from zdb_tally('so_posts', 'last_activity_date', 'week:-1d', 'cheese', 5000, 'term');