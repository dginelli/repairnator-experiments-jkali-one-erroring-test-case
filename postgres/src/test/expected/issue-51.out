select * from zdb_tally('so_posts', 'last_activity_date', 'week', 'cheese', 5000, 'term');
    term    | count 
------------+-------
 2009-07-20 |     1
 2010-10-11 |     1
 2010-11-08 |     1
 2010-11-29 |     1
 2010-12-06 |     1
 2010-12-13 |     1
 2010-12-27 |     1
 2011-01-31 |     1
 2011-02-21 |     1
 2011-03-07 |     1
 2011-03-21 |     1
 2011-04-25 |     1
 2011-06-13 |     1
 2011-07-04 |     1
 2011-08-15 |     1
 2011-09-19 |     1
 2011-10-03 |     1
 2011-11-21 |     1
 2011-12-05 |     1
 2011-12-12 |     1
 2012-02-06 |     2
 2012-03-05 |     2
 2012-05-07 |     1
 2012-05-14 |     1
 2012-05-21 |     1
 2012-08-13 |     1
 2012-09-10 |     1
 2012-11-12 |     1
 2013-01-21 |     1
 2013-01-28 |     2
 2013-02-11 |     1
 2013-03-04 |     1
 2013-05-20 |     1
 2013-12-16 |     1
 2013-12-23 |     1
 2014-01-20 |     1
 2014-06-30 |     1
 2014-08-18 |     1
 2014-09-15 |     1
 2014-09-29 |     1
 2014-10-06 |     1
 2014-11-24 |     1
 2014-12-15 |     2
 2015-02-09 |     1
(44 rows)

select * from zdb_tally('so_posts', 'last_activity_date', 'week:-1d', 'cheese', 5000, 'term');
    term    | count 
------------+-------
 2009-07-19 |     1
 2010-10-10 |     1
 2010-11-07 |     1
 2010-11-28 |     1
 2010-12-05 |     1
 2010-12-12 |     1
 2010-12-26 |     1
 2011-01-30 |     1
 2011-02-20 |     1
 2011-03-06 |     1
 2011-03-20 |     1
 2011-04-24 |     1
 2011-06-12 |     1
 2011-07-03 |     1
 2011-08-14 |     1
 2011-09-18 |     1
 2011-10-02 |     1
 2011-11-27 |     1
 2011-12-04 |     1
 2011-12-11 |     1
 2012-02-05 |     2
 2012-03-04 |     1
 2012-03-11 |     1
 2012-05-06 |     1
 2012-05-13 |     1
 2012-05-20 |     1
 2012-08-19 |     1
 2012-09-16 |     1
 2012-11-11 |     1
 2013-01-20 |     1
 2013-01-27 |     2
 2013-02-10 |     1
 2013-03-03 |     1
 2013-05-19 |     1
 2013-12-15 |     1
 2013-12-22 |     1
 2014-01-19 |     1
 2014-06-29 |     1
 2014-08-17 |     1
 2014-09-14 |     1
 2014-09-28 |     1
 2014-10-05 |     1
 2014-11-23 |     1
 2014-12-14 |     1
 2014-12-21 |     1
 2015-02-15 |     1
(46 rows)

